### ISSUE TITLE
I can't open live server

### ISSUE THREAD

**Issue by Deotyma** [ritwickdey/vscode-live-server#204]
Hello,
I just install Live Server for Visual Studio Code but il doesn't work. There is no go live button and can't open my html file directly with Live Server. I work on OS Sierra. 
Thanks for any help.
Ewa

**Comment by MariahAdams** [comment]
I am currently experiencing the same issue, and have been for some time. I have tried multiple time to uninstall/reinstall, disable/enable, reload the extension with no success. This started happening after one of the last VSCode updates.

Mac High Sierra, VSCode Version 1.28.2

**Comment by ddyson** [comment]
Yep, fresh install of latest VS Code and Live Server and it just doesn't work. Not a good first time experience. Are there alternatives out there?

**Comment by Sniper1219** [comment]
Same problem for me. :(

**Comment by Markxuxiao** [comment]
![image](https://user-images.githubusercontent.com/9879038/54742268-e4dbdf00-4bfb-11e9-871a-63c3c6b67235.png)
Attention please
